terror/terror.go

```diff
@@ -328,6 +328,14 @@ func MustNil(err error) {
 	}
 }
 
+// CleanNotNil cleans up and fatals if err is not nil.
```

Comment:
Why put it in the terror package?

Reply:
How about we reuse the `MustNil` function, and change it like
```
func MustNil(err error, closeFn ...func())
```

```